src/vscode-bicep/src/test/e2e/commands.ts

```diff
@@ -98,3 +98,18 @@ export async function executeForceModulesRestoreCommand(
     documentUri
   );
 }
+
+export async function executePasteAsBicepCommand(
+  documentUri: vscode.Uri
+): Promise<void> {
+  return await vscode.commands.executeCommand(
+    "bicep.pasteAsBicep",
+    documentUri
+  );
+}
+
+export async function executeEditorPaste(): Promise<void> {
```

Comment:
nit: Add `Command` suffix for sake for consistency?

Reply:
done